lib/tasks/publishing_api.rake

```diff
@@ -34,47 +34,18 @@ namespace :publishing_api do
   end
 
   desc "Send publishable item links to Publishing API."
-  task :publishing_api_patch_links, [:item_classes] => :environment do |_, args|
-    def patch_links(item)
-      retries = 0
-      begin
-        content_id = item.content_id
-        links = PublishingApiPresenters.presenter_for(item).links
-        if links && !links.empty?
-          Whitehall.publishing_api_v2_client.patch_links(content_id, {links: links})
-        end
-      rescue GdsApi::TimedOutException, Timeout::Error
-        retries += 1
-        if retries <= 3
-          $stderr.puts "Class #{item.class} id: #{item.id} Timeout: retry #{retries}"
-          sleep 0.5
-          retry
-        end
-        raise
-      end
-    rescue => err
-      $stderr.puts "Class: #{item.class}; id: #{item.id}; Error: #{err.message}"
-    end
-
-    args[:item_classes].split(',').each do |class_name|
-      klass = class_name.constantize
-      if klass.ancestors.include?(Edition)
-        editions = klass.published
-        count = editions.count
-        $stdout.puts "# Sending #{count} published #{class_name} items to Publishing API"
-      else
-        editions = klass.all
-        count = editions.count
-        $stdout.puts "# Sending all #{count} #{class_name} items to Publishing API"
-      end
+  task publishing_api_patch_links: :environment do
+    editions = Edition.published
+    count = editions.count
+    $stdout.puts "# Sending #{count} published editions to Publishing API"
 
-      editions.find_each.with_index do |publishable_item, i|
-        patch_links(publishable_item)
+    editions.select(:id).find_each.with_index do |item_id, i|
```

Comment:
I think calling `#select` on a relation still returns an `ActiveRecord` object. Perhaps [`#pluck`](http://apidock.com/rails/ActiveRecord/Calculations/pluck) is what you're looking for here.